### PR TITLE
Slight improvement to the not runnable message.

### PR DESCRIPTION
**Pull request by livarcocc**
Fixes https://github.com/dotnet/cli/issues/2859 and https://github.com/dotnet/cli/issues/5111

@dotnet/dotnet-cli

### CODE REVIEW

**Comment by livarcocc** [comment]
@dotnet-bot Test Debian8.2 x64 Debug Build

**Comment by livarcocc** [comment]
@dotnet-bot Test Windows_NT x86 Debug Build

**Comment by livarcocc** [comment]
@dotnet-bot Test OSX10.12 x64 Release Build

**Comment by livarcocc** [comment]
@MattGertz for approval. This is just a string change.

File: src/dotnet/commands/dotnet-run/LocalizableStrings.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ internal class LocalizableStrings
 
         public const string RunCommandExceptionUnableToRunSpecifyFramework = "Unable to run your project\nYour project targets multiple frameworks. Please specify which framework to run using '{0}'.";
 
-        public const string RunCommandExceptionUnableToRun = "Unable to run your project\nPlease ensure you have a runnable project type and ensure '{0}' supports this project.\nThe current {1} is '{2}'";
+        public const string RunCommandExceptionUnableToRun = "Unable to run your project.\nPlease ensure you have a runnable project type and ensure '{0}' supports this project.\nA runnable project should target a runnbale TFM (for instance, netcoreapp2.0) and have OutputType 'Exe'.\nThe current {1} is '{2}'.";
```
Comment (akoeplinger) [inline src/dotnet/commands/dotnet-run/LocalizableStrings.cs]: typo: "runnbale TFM"

Comment (livarcocc) [inline src/dotnet/commands/dotnet-run/LocalizableStrings.cs]: Good catch.

**Pull request merged**